cocos/ui/UIEditBox/UIEditBoxImpl-common.cpp

```diff
@@ -95,8 +99,39 @@ void EditBoxImplCommon::initInactiveLabels(const Size& size)
 
 void EditBoxImplCommon::placeInactiveLabels()
 {
-    _label->setPosition(CC_EDIT_BOX_PADDING, _contentSize.height / 2.0f);
-    _labelPlaceHolder->setPosition(CC_EDIT_BOX_PADDING, _contentSize.height / 2.0f);
+    switch (_textHorizontalAlignMent) {
+        case cocos2d::TextHAlignment::LEFT :
```

Comment:
@anjandaffo 
I think here we should call `label->setTextAlignment` API, otherwise the alignment is wrong after finishing typing.
A quick way to reproduce this issue:

```
 _editName->setVerticalTextAlignment(TextVAlignment::TOP);
        _editName->setHorizontalTextAlignment(TextHAlignment::RIGHT);

```